crates/next-core/src/next_server/resolve.rs

```diff
@@ -47,8 +90,12 @@ impl ResolvePlugin for ExternalCjsModulesResolvePlugin {
             return Ok(ResolveResultOptionVc::none());
         }
 
-        // mjs -> esm module
-        if Some("mjs") == raw_fs_path.extension() {
+        // node.js only supports these file extensions
+        // mjs is an esm module and we can't bundle that yet
+        if !matches!(
+            raw_fs_path.extension(),
+            Some("cjs" | "js" | "node" | "json")
+        ) {
```

Comment:
Could we get rid of this and just use `is_node_resolveable`?

Reply:
Seems like no:

* `package/style.css` is node.js resolveable, but node.js will still struggle with it.
* It's a very cheap check. `is_node_resolveable` is very expensive since it need to resolve the whole request.
* We also want to avoid `.mjs` until we support ESM.